Source/Mosa.DeviceDriver/PCI/VirtIO/VirtIODevice.cs

```diff
@@ -38,30 +38,30 @@ public class VirtIODevice
 	public VirtIODevice(Device device)
 	{
 		devName = device.Name;
-		pciDevice = (PCIDevice)device.Parent.DeviceDriver;
+		configuration = (PCIDeviceConfiguration)device.Configuration;
 
-		if (pciDevice.RevisionID < 1)
+		if (configuration.RevisionID < 1)
 		{
-			HAL.DebugWriteLine("[" + devName + "] pci revision id mismatch; abort");
+			HAL.DebugWriteLine($"[{devName}] pci revision id mismatch; abort");
```

Comment:
Be careful; Rosyln may choose an implementation that is not supported.

Reply:
Huh, anything I could read on for that? It seems weird that it would choose multiple implementations for 1 feature. Either way, can't we just implement those?

Reply:
It's caused random crashes in the past. However, if everything works, I'll approve. Let's keep an eye on it, however.